vllm/model_executor/model_loader/weight_utils.py

```diff
@@ -319,17 +321,23 @@ def kv_cache_scales_loader(
             return layer_scales_map.items()
 
     except FileNotFoundError:
-        logger.error("File or directory '%s' not found.", filename)
+        logger_data = {"filename": filename}
+        logger.error("File or directory '%s' not found.", filename, extra=logger_data)
     except json.JSONDecodeError:
-        logger.error("Error decoding JSON in file '%s'.", filename)
+        logger_data = {"filename": filename}
+        logger.error("Error decoding JSON in file '%s'.", filename, extra=logger_data)
     except Exception as e:
-        logger.error("An error occurred while reading '%s': %s", filename, e)
+        logger_data = {"filename": filename}
+        logger.error(
+          "An error occurred while reading '%s': %s", filename, e, extra=logger_data
+        )
     # This section is reached if and only if any of the excepts are hit
     # Return an empty iterable (list) => no KV cache scales are loaded
     # which ultimately defaults to 1.0 scales
     logger.warning(
         "Defaulting to KV cache scaling factors = 1.0 for all "
         "layers in TP rank %d as an error occurred during loading.", tp_rank)
+    
```

Comment:
```suggestion
```

Reply:
Or maybe there was `logger_data` you'd intended to add here? I'm not sure if it is worth it